packages/api/tests/api/videos.test.ts

```diff
@@ -0,0 +1,108 @@
+// add tests to ensure videos are being returned properly
+import { videos } from '../../src/api/videos';
+import logger from '../../src/logger';
+import { testHandler } from '../testUtils/testHandler';
+
+
+// Video object mocks
+interface Video {
+  id: string;
+  title: string;
+  durationInSeconds: number;
+  url: string;
+}
+
+const video1 : Video = {
+  id: '1',
+  title: 'video1',
+  durationInSeconds: 102,
+  url: "www.test.org"
+}
+
+const video2 : Video = {
+  id: '2',
+  title: 'video2',
+  durationInSeconds: 1109,
+  url: "www.test2.org"
+}
+
+const loggerSpy = jest.spyOn(logger, 'error').mockImplementation();
+
+describe('/videos', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  it("returns all videos when videos", async () => {
+    const expectedVideos = [video1, video2];
+    const handler = testHandler(videos);
+    handler.entityManager.find.mockResolvedValue(expectedVideos);
```

Comment:
Make sure that you are mocking it `once` however many times you want to so you can make sure it doesn't carry over to other tests. (I'll also put suggestions for the other ones)

```suggestion
    handler.entityManager.find.mockResolvedValueOnce(expectedVideos);
```